Objects/longobject.c

```diff
@@ -5402,6 +5402,9 @@ int_from_bytes_impl(PyTypeObject *type, PyObject *bytes_obj,
         (unsigned char *)PyBytes_AS_STRING(bytes), Py_SIZE(bytes),
         little_endian, is_signed);
     Py_DECREF(bytes);
+    if (long_obj == NULL) {
```

Comment:
This check can be merged with the following check.
```c
if (long_obj != NULL && type != &PyLong_Type) {
```

Reply:
Sure.